examples/00_quick_start/sasrec_amazon.ipynb

```diff
@@ -37,8 +37,10 @@
      "name": "stderr",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #15.        return df
</pre><p>I would suggest clarifying that this function is only guaranteed to give the desired result after a sufficient number of iterations. I.e. you may still get some users or items with &lt; min_rating ratings and may need to continue for more iterations.</p><p>The problem looks like an NP hard one and is equivalent to finding the biggest subgraph of minimum degree.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/microsoft/recommenders/pull/1621/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>

Reply:
<p>I have modified the function so that filtering continues until the min_rating conditions are satisfied for both the users and items.  </p>